### PR TITLE
zzuf: 0.14

### PR DESCRIPTION
**Pull request by denji**

### CODE REVIEW

File: Library/Formula/zzuf.rb [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
-  # This has been fixed upstream and should be included in the next release.
-  patch :p3 do
-    url "https://trac.macports.org/export/78051/trunk/dports/security/zzuf/files/patch-src-libzzuf-lib--mem.c.diff"
-    sha1 "bc40649d8bdd589ef3876796d4791f28220d964a"
+  stable do
+    url "https://github.com/samhocevar/zzuf/releases/download/v0.14/zzuf-0.14.tar.gz"
+    sha256 "291b41d53ae6df75d0d2b4a0faa333adfc4a02b9ca8706bee47ef7be657966ce"
+    # Fix missing semi-colon so compile succeeds
+    # https://github.com/x9prototype/zzuf/pull/1
+    patch :DATA
```
Comment (mikemcquaid) [inline Library/Formula/zzuf.rb]: Please use `patch do` and `https://github.com/x9prototype/zzuf/pull/1.patch`

**Comment by dunn** [comment]
Needs rebasing.

**Comment by dunn** [comment]
@BrewTestBot test this please

**Comment by denji** [comment]
@dunn it's fine now?

**Comment by dunn** [comment]
Looks like it, but we left it too long so the bottles expired :(

**Comment by dunn** [comment]
@BrewTestBot test this please

**Comment by dunn** [comment]
@BrewTestBot test this please

**Comment by dunn** [comment]
Sorry for the delay on this; merged in 9815d004af2064876ed450e3788483cc0c4e068f.

**Pull request closed without merging**